samples/RefreshFlow/RunDemo.ps1

```diff
@@ -0,0 +1,40 @@
+                                                          
+$global:p = @();                                  
+                                                          
+function global:Find-ChildProcess {
+  param($ID=$PID)
+
+  $result = Get-CimInstance win32_process | 
+    where { $_.ParentProcessId -eq $ID } 
+    select -Property ProcessId 
+
+  $result
+  $result | 
+    Where-Object { $_.ProcessId -ne $null } | 
+    ForEach-Object {
+      Find-ChildProcess -id $_.ProcessId
+    }
+}
+
+function global:Kill-Demo {
+  $Global:p | 
+    foreach { Find-ChildProcess -ID $_.Id } | 
+    foreach { kill -id $_.ProcessId }
+}
+
+[System.Environment]::SetEnvironmentVariable("ASPNETCORE_ENVIRONMENT", "Development");
+
+# Authorization Server
+Push-Location "./AuthorizationServer"
+dotnet restore
+dotnet build --no-incremental #rebuild
+$global:p += Start-Process dotnet -ArgumentList "run server.urls=http://localhost:5056" -PassThru
+Pop-Location
+
+# Angular Application
+Push-Location "./AngularApp"
+npm install -y
+$global:p += Start-Process ng -ArgumentList "serve --port 5055" -PassThru
```

Comment:
Instead of `ng` use ` node_modules\.bin\ng` because the package.json is installing the Angular CLI locally not globally.